CONTRIBUTING.md

```diff
@@ -11,6 +11,11 @@ Instead, modify the sources in the ```www``` folder and compile a new version of
 
 ```node.js``` and ```npm``` are needed for building this project. ```npm``` comes bundled with the ```node.js``` installer. You can download the ```node.js``` installer here: https://nodejs.org/download/. 
 
+To run Android tests, make sure you have ```sdk\tools``` and  ```sdk\platform-tools``` in your PATH.
+
+If you would like the tests to start the emulators for you, configure an Android emulator using the AVD and name it ```emulator```.
```

Comment:
I think emulator names should be passed by command line with defaults if not provided, like what Daniel suggests. As to the test task names (android, ios+ui, ios+wk etc..) I will leave it up to you since it looks like its going to be a huge undertaking to refactor everything